.gitignore

```diff
@@ -103,3 +103,4 @@ _build_endpoint/
 endpoint.*.update
 .repo
 legato/
+endpoint/build
```

Comment:
Why did you put the generated files in directory `endpoint/build`? Can you put them in the consistent way?

Reply:
The endpoint build is the temporary directory for such as flatcc cmake generated file, OBD2 headers, and flatbuffer serializer headers(compile by flatcc). You mean put them into legato?

Reply:
If a dedicated `build` directory exists at the top-level directory, then all the generated files should appear there.